lib/project-view.coffee

```diff
@@ -37,6 +45,7 @@ module.exports = ProjectView =
         @subscribeUpdateEvents()
         # Initally update the root names
         @updateRoots(@treeView.roots)
+      @regex = new RegExp(atom.config.get('project-view.regexMatch')
```

Comment:
Missing trailing `)` character at the end of the line? Also setting the `@regex` property should be happening before calling `@updateRoots`.